PAYMENTS.md

```diff
@@ -0,0 +1,27 @@
+# Integrate your Stripe account to accept your user's payments
+All the information that you need to setup your Stripe account into OSEM.
+If you have any problems with installing don't hesitate to [contact us](https://github.com/openSUSE/osem#contact)
+
+## Configure Stripe into the application
+To configure Stripe into your application all you need to do is add the private and publishable keys into the Rails environment.
+
+You can add the your keys in `dotenv.example` and in `config/secrets.yml` into these variables:
+
+If you are using the application in development mode your config should look like this:  
+  `STRIPE_PUBLISHABLE_KEY = 'pk_test_random123example456'`  
+  `STRIPE_SECRET_KEY = 'sk_test_random123example456'`
+
+Otherwise, while in production mode, it should look like:  
```

Comment:
Stripe provides keys in different forms:
Production key: sk/pk_**live**_random123example456
Test key: sk/pk_**test**_random123example456